testing/python/language/test_rand.py

```
@@ -0,0 +1,37 @@
+import tilelang
+import tilelang.language as T  # noqa: N812
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `N812`)

Remove unused `noqa` directive

(RUF100)